website/content/docs/about-vault/why-use-vault/identities.mdx

```diff
@@ -0,0 +1,126 @@
+---
+layout: docs
+page_title: Manage identities and authentication
+description: >-
+  Control client access to sensitive information with managed entities, identity
+  tokens, OIDC workflows, and WIF
+---
+
+# Manage identities and authentication
+
+@include '/why-use-vault/manage-identities-small.mdx'
+
+
+## Manage identities and entities
+
+Vault provides centralized identity management through the identity plugin so
+clients can use accounts with different identity providers to authenticate to
+Vault. The identity plugin ties authentication instances to a single,
+consolidated representation called an **entity**. Entities map to aliases for
+corresponding accounts with authentication providers and policies that authorize
+those entities to take action within Vault.
+
+
+<Tabs>
+<Tab heading="Key concepts + Overviews" group="overviews">
+
+- [Identities overview](/vault/docs/concepts/identity)
+- [Policies overview](/vault/docs/concepts/policies)
+- [Authentication overview](/vault/docs/concepts/auth)
+- [Clients and entities overview](/vault/docs/concepts/client-count)
+
+
+</Tab>
+<Tab heading="Guides" group="guides">
+
+- [Find and resolve duplicate Vault identities](/vault/docs/secrets/identity/deduplication)
+- [Use identity tokens](/vault/docs/secrets/identity/identity-token)
+- [Identity plugin overview](/vault/docs/secrets/identity)
+
+</Tab>
+<Tab heading="Tutorials" group="tutorials">
+
+- [Introduction to tokens](/vault/tutorials/get-started/introduction-tokens)
+- [Introduction to roles](/vault/tutorials/get-started/introduction-roles)
+- [Introduction to policies](/vault/tutorials/get-started/introduction-policies)
+- [Implement identity entities and groups](/vault/tutorials/auth-methods/identity)
```

Comment:
Adding AppRole tutorial as an example to apply the knowledge about token, role, policy, and identity.

```suggestion
- [Introduction to tokens](/vault/tutorials/get-started/introduction-tokens)
- [Introduction to roles](/vault/tutorials/get-started/introduction-roles)
- [Introduction to policies](/vault/tutorials/get-started/introduction-policies)
- [Implement identity entities and groups](/vault/tutorials/auth-methods/identity)
- [Generate tokens for machine authentication with AppRole](/vault/tutorials/auth-methods/approle)
```